cpg-core/src/testFixtures/kotlin/de/fraunhofer/aisec/cpg/frontends/TestLanguage.kt

```diff
@@ -39,11 +39,15 @@ import kotlin.reflect.KClass
  * This is a test language that can be used for unit test, where we need a language but do not have
  * a specific one.
  */
-class TestLanguage : Language<TestLanguageFrontend>() {
+class TestLanguage(namespaceDelimiter: String) : Language<TestLanguageFrontend>() {
```

Comment:
Can we make `::` the default at least? So we can just use an empty constructor, if wanted.